test/evergreen.yml

```diff
@@ -240,17 +241,22 @@ functions:
               (cd lang/python && make ../../../lang/python/wiredtiger_wrap.c)
             fi
 
-            (cd ../dist && sh s_docs)
-            (cd .. && rm -rf docs-$branch && mv docs docs-$branch)
+            (cd ../dist && sh s_docs && echo "The documentation for $branch was successfully generated.")
+            # Save generated documentation
+            (cd .. && mv docs docs-$branch)
           done
+
   "update wiredtiger docs":
     - command: shell.exec
       params:
-        working_dir: "wiredtiger"
         shell: bash
         script: |
+          # Use a single function to update the documentation of each supported WiredTiger branch.
+          # This is useful as not all branches have a dedicated Evergreen project. Furthermore, the
+          # documentation-update task is not triggered by every commit, we rely on the activity of
+          # the develop branch to update the documentation of all supported branches.
+          # The verbose option is not set on purpose to avoid showing credentials.
```

Comment:
I'm not sure if explicitly commenting "avoid showing credentials" in our codebase could be a security concern. I'd prefer not to mention it, and simply use the "silent" parameter which is self-descriptive.

Reply:
See 15dea4d0e62cb751be2a84a8d62199ccfcfe4360